commit 3b9e1e5cf08481ddc5fde4095acea184568fd6d7
Author: Greg Banks <gnb@fmeh.org>
Date:   Fri Jan 25 20:28:56 2013 +1100

    Don't build docs during install:
    
    as the dist tarball already contains them, we can just install them.
    This avoids having to run Doxygen on build machines, as it seems to be
    buggy on some releases of Ubuntu.

diff --git a/Makefile b/Makefile
index ccce497..555dce5 100644
--- a/Makefile
+++ b/Makefile
@@ -36,8 +36,8 @@ INSTALL=	/usr/bin/install -c
 RANLIB=		ranlib
 depdir=		.deps
 
-SUBDIRS_PRE=
-SUBDIRS_POST=	tests doc/get-start
+SUBDIRS_PRE=	doc/get-start
+SUBDIRS_POST=	tests
 
 all clean distclean check install docs:
 	@for dir in $(SUBDIRS_PRE) ; do $(MAKE) -C $$dir $@ ; done
@@ -167,11 +167,12 @@ DOC_DELIVERABLES= \
 docs-local: Doxyfile
 	$(RM) -r doc/api-ref doc/man
 	doxygen
+	$(RM) doc-$(VERSION)
 	ln -s doc doc-$(VERSION)
 	tar -chjvf doc-$(VERSION).tar.bz2 $(addprefix doc-$(VERSION)/,$(DOC_DELIVERABLES))
 	$(RM) doc-$(VERSION)
 
-install: docs
+install:
 
 install-local:
 	$(INSTALL) -d $(DESTDIR)$(includedir)/novaprova/np
